cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -946,33 +944,14 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
-  @Test def use_index_with_distance_query_composite() {
-    executePreparationQueries(List("CREATE BTREE INDEX FOR (p:Person) ON (p.place,p.name)"))
-    executePreparationQueries(
-      (for(x <- -10 to 10; y <- -10 to 10) yield s"CREATE (:Person {place: point({x:$x, y:$y}), name: '${x+y}' } )").toList)
-    profileQuery(
-      title = "Spatial distance searches (composite index)",
```

Comment:
I'm guessing this test is called from somewhere to be included in the documentation, you need to remove it from there as well (goes for all removed tests)